transformers/storage/cat/repository_test.go

```diff
@@ -229,7 +234,8 @@ var _ = Describe("Cat storage repository", func() {
 				Metadata:      storage.GetValueMetadata(cat.IlkLump, map[storage.Key]string{constants.Ilk: test_helpers.FakeIlk.Hex}, storage.Uint256),
 				PropertyName:  "Lump",
 				PropertyValue: strconv.Itoa(rand.Int()),
-				TableName:     "maker.cat_ilk_lump",
```

Comment:
I was thinking of making a similar change in some of the other places we're hardcoding table names, but I'm not sure if that would reduce the readability of queries... ie. line 204
```			
err = db.Get(&result, `SELECT diff_id, header_id, ilk_id AS key, lump AS value FROM maker.cat_ilk_lump`)
```

could become
```
err = db.Get(&result, `SELECT diff_id, header_id, ilk_id AS key, lump AS value FROM $1`,
     shared.GetFullTableName(constants.MakerSchema, constants.CatIlkLump)
)

```

What do you all think?

Reply:
👍 I don't think it impacts the readability that much. After `FROM` my brain just thinks "..what table is this coming `FROM`?"

Reply:
I'm a fan of the change - to me the readability concern is outweighed by the scope of change entailed by updating a table name 👍